web/src/styles/scss/layout.scss

```diff
@@ -23,6 +23,7 @@
   float: left;
   + .site-main {
     margin-left: 244px; // sidebar width plus $spacer-4
+    min-height: 1100px; // min height of main content to prevent menu from pushing up under the header
```

Comment:
I'm curious where this `1100px` comes from, and if maybe it would be better as a `calc`ed value? Something similar to the sidebar's max height, for instance: `min-height: calc(100vh - 60px);`

Reply:
I added the following comment to the CSS file to explain my reasoning:

The state icon above the menu will push up under the header if the right side of the page is too short.  So we need a min-height for site-main to be at least the height of the menu (plus a little).  Note: "Private Contractor costs" is a particularly short page where the effect can be seen.  Since there isn't an easy way to do relative heights of "unrelated" elements in CSS, just use a constant.  Remember to adjust it when adding new menu items that change the menu size.